### DISCUSSION TITLE
How to run an alter migration?

### DISCUSSION THREAD

**Discussion by arferreira** [loco-rs/loco#146] [Q&A]
Hello folks, 
I was trying to add a field on the user model. Is there a command to run a migration to alter a table?

Thanks,

**Comment by jondot** [comment]
Hi,
If you just started out, you can change the User migration, and re-create the database.
For projects where you cannot recreate database, you can create a new migration, and then use schema mutation:

https://www.sea-ql.org/SeaORM/docs/migration/writing-migration/#schema-mutation-methods

**Comment by jondot** [comment]
@arferreira there is now a built-in way to do this, as well as docs https://loco.rs/docs/the-app/migration/

**Reply by arferreira** [reply]
@jondot that's awesome. How could I update my current loco app to get an updated version?
